src/app/docs/alert/alert-docs.component.html

```diff
@@ -4,10 +4,269 @@
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Danger alert – for errors or to let the user know the system is in an unusable state.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-2.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Info alert – for messages that provide helpful information or context but do not require action.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-3.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Success alert – for messages conveying a positive or successful state.
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td class="sky-docs-table-cell">
+              <sky-docs-design-guideline-thumbnail
+              imageSource="~/assets/img/guidelines/alert/alert-type-4.png"
+              >
+              </sky-docs-design-guideline-thumbnail>
+            </td>
+            <td class="sky-docs-table-cell">
+              <p>
+                Warning alert – for messages the user should know about and perhaps act on, but that do not constitute an error state or indicate that something is wrong.
+              </p>
+            </td>
+          </tr>
+        </table>
+      </sky-docs-design-guideline>
+        
+
+      <sky-docs-design-guideline
+        heading="Close button"
+      >
+        <p>
+          If the user should be able to dismiss the alert from the page after reading, you can include an optional close button.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-option-1.png"
+        >
+        </sky-docs-design-guideline-thumbnail>
+        <p>
+          Do not make an alert dismissible if it is important for users see every time or if it is triggered by underlying data.
+        </p>
+      </sky-docs-design-guideline>
+    </sky-docs-demo-page-section>
+
+    <sky-docs-demo-page-section
+      heading="Content"
+    >
+      <sky-docs-design-guideline>
+        <p>
+          Text inside an alert can contain hyperlinks. For accessibility reasons, these links will not be styled blue as other links in the system are.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+          imageSource="~/assets/img/guidelines/alert/alert-usage-2.png"
```

Comment:
Already noted the text suggestions for this image above, but just pointing out that the same image is used here to make sure that's correct since we use three different versions of the preceding image.